packages/livekit-rtc/src/utils.ts

```diff
@@ -0,0 +1,25 @@
+// SPDX-FileCopyrightText: 2024 LiveKit, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+export function diffAttributes(
```

Comment:
Ah awesome!
cc @keepingitneil, we should do the same for https://github.com/livekit/python-sdks/pull/217